_projects/rust-sdk-python-bindings

```diff
@@ -0,0 +1,41 @@
+---
+name: Matrix-Rust-SDK-Python
+desc: Building Python/nio bindings for using the matrix rust sdk
+requirements:
+# Student requirements:
+ - Worked with Rust FFI
+ - Familiar with building Python Bindings
+difficulty: low
```

Comment:
I would probably rate this medium, you need to be proficient in two languages and there's quite a lot to do, bindings, docs, integration into an existing project...

Reply:
```suggestion
difficulty: medium
```